kibana/build.yml

```diff
@@ -2,7 +2,7 @@ repository: monasca/kibana
 variants:
   - tag: latest
     aliases:
-      - :4.6.3-master
+      - :4.6.4-master
```

Comment:
That is actually not true, this particular case is a Kibana version itself set to be 4.6.3. That said, I guess you need to have
```!push kibana``` to update master image directly.

@timothyb89 is that true ?